internal_worker.go

```diff
@@ -757,13 +767,15 @@ func newAggregatedWorker(
 ) (worker Worker) {
 	wOptions := fillWorkerOptionsDefaults(options)
 	workerParams := workerExecutionParameters{
-		TaskList:                  taskList,
-		ConcurrentPollRoutineSize: defaultConcurrentPollRoutineSize,
-		Identity:                  wOptions.Identity,
-		MetricsScope:              wOptions.MetricsScope,
-		Logger:                    wOptions.Logger,
-		EnableLoggingInReplay:     wOptions.EnableLoggingInReplay,
-		UserContext:               wOptions.BackgroundActivityContext,
+		TaskList:                        taskList,
+		ConcurrentPollRoutineSize:       defaultConcurrentPollRoutineSize,
```

Comment:
add method header comments on the default values for the pollerCount/concurrentTask count etc. Also would be cool to provide a rough estimation of the task throughput they can achieve with the default values and when they should consider changing this value.